pkgs/gohugoio/hugo-extended/registry.yaml

```diff
@@ -0,0 +1,47 @@
+packages:
+  - type: github_release
+    repo_owner: gohugoio
+    repo_name: hugo
+    name: gohugoio/hugo-extended
```

Comment:
```suggestion
    name: gohugoio/hugo/hugo-extended
```

The package name should start with `gohugo/hugo/` to update the package by Renovate.

Then the package should be moved to pkgs/gohugoio/hugo/hugo-extended.